### ISSUE TITLE
checkpoints unavailable

### ISSUE THREAD

**Issue by Herice31-53** [MorenoLaQuatra/bart-it#2]
Hello, 
This seems to be very promising work, and I am looking forward to testing the summarization pipeline. However if bart-it seems to be available with transformers, it is not the case with any of the fine tuned models or any of the tokenizers. Is there another way to load the bart-it-fanpage for example ?
Thank you in advance!

**Comment by MorenoLaQuatra** [comment]
Hi @Herice31-53 ,

Thank you for the kind words. You can access all models by simply using the following:

```python
from transformers import AutoModel, AutoTokenizer

model_tag = "morenolq/bart-it"
model = AutoModel.from_pretrained(model_tag)
tokenizer = AutoTokenizer.from_pretrained(model_tag)
```

`model_tag` can be set to:
- [`morenolq/bart-it`](https://huggingface.co/morenolq/bart-it) - for pre-trained model (no fine-tuning)
- [`morenolq/bart-it-fanpage`](https://huggingface.co/morenolq/bart-it-fanpage) - for [fanpage](https://www.fanpage.it/)
- [`morenolq/bart-it-ilpost`](https://huggingface.co/morenolq/bart-it-ilpost) - for [Il post](https://www.ilpost.it/)
- [`morenolq/bart-it-WITS`](https://huggingface.co/morenolq/bart-it-WITS) - for Wikipedia style summarization

If you want to access one of them independently, you can clone the repository by running:

```bash
git clone URL
```

Where `URL` is simply one of the URLs on the HuggingFace Hub (see links above).

**Comment by Herice31-53** [comment]
Hello,

This is actually my issue.
The code you provided:

from transformers import AutoModel, AutoTokenizer
model_tag = "morenolq/bart-it"
model = AutoModel.from_pretrained(model_tag)

works only for model_tag=[morenolq/bart-it](https://huggingface.co/morenolq/bart-it) 
When calling this with any other model_tag, the model simply won't load. I checked and they are indeed present on Huggingface since I do not have any error, but the model still won't load.
Any idea why ?
Worst case I can just retrain them using your summarization training.

Thanks for your reply.

Edit for those who may have the same problems: it was a library issue. I cannot tell which one, but by "git clone URL", then creating a new environment with python 3.8, torch 2.0.1 and transformers 4.30.2; my issue got solved.
